src/themes/playful.css

```
@@ -334,14 +350,404 @@
   --wa-form-controls-required-content-color: inherit;
   --wa-form-controls-required-content-offset: -0.1em;
 
-  --wa-panel-corners: var(--wa-corners-m);
+  --wa-panel-corners: var(--wa-corners-l);
```

```suggestion
  --wa-panel-corners: var(--wa-corners-m);
```
If we make an adjustment to the `corners` properties above, we can keep using `--wa-corners-m` here. We can reserve `--wa-corners-l` for just the product cards since they have rounder corners than any of our other elements (alerts, message composer, support table, etc.).